rust/src/wrapper/att.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Support for the ATT layer.
+
+pub use crate::internal::att::{
+    AttributePermission, AttributePermissions, AttributeRead, AttributeUuid, AttributeWrite,
+    MutexRead, NoOpWrite,
+};
+use crate::wrapper::core::TryToPy;
+use pyo3::prelude::PyModule;
+use pyo3::types::PyBytes;
+use pyo3::{intern, PyAny, PyResult, Python};
+
+impl TryToPy for AttributeUuid {
```

Comment:
Not quite sure how it'll work out, but I'm experimenting with having `Try[To|From]Py` in the `wrapper` module distinct from the relatively pure Rust structs in `crate::internal`